EpsilonCCS/Src/TelemetryReporting/TelemetryReporting.c

```diff
@@ -7,10 +7,15 @@
 #include "KeyMotorData.h"
 #include "MotorDetailsData.h"
 #include "DriverControlData.h"
+#include "MotorFaultsData.h"
+#include "BatteryFaultsData.h"
 
 #define KEY_MOTOR_LENGTH (43)
 #define MOTOR_DETAILS_LENGTH (69)
 #define DRIVER_CONTROLS_LENGTH (9)
+#define MOTOR_FAULTS_LENGTH (9)
+#define BATTERY_FAULTS_LENGTH (3)
```

Comment:
KEY_MOTOR_LENGTH is wrong in the protocol, it should just be 43 bytes long.

Battery faults length is correct within the protocol. This reasoning is 1 byte for package ID, 4 bytes for battery faults (21 faults which mean 21 bits. With 8 bits in a byte that is at minimum 3 bytes. For alignment and general ease that is just rounded up to 4 bytes). 2 Bytes for the Limit flags (16 flags, 16 bits, 2 bytes)